CONTRIBUTING.rst

```diff
@@ -359,11 +252,6 @@ Pull requests
    (``- [ ] take over the world``) to let others know what more to
    expect in the near future.
 
--  There are a number of emoji that have specific meanings within our
-   github conversations. The most important one is :dancer: which means
```

Comment:
is it because there is no longer a dancer emoji and only 👯 :). Fun concept but probably hard to get everyone to commit to it.

Reply:
It's more that there is actually a approve button now so we don't need to pick a emoji but good question what happened to the dancer ❓